### ISSUE TITLE
Update the buildToolsVersion

### ISSUE THREAD

**Issue by Qdafengzi** [mixpanel/mixpanel-react-native#259]
```
The specified Android SDK Build Tools version (30.0.3) is ignored, as it is below the minimum supported version (34.0.0) for Android Gradle Plugin 8.4.1.
Android SDK Build Tools 34.0.0 will be used.
To suppress this warning, remove "buildToolsVersion '30.0.3'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.
```

Can modify like this ?

  compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
  buildToolsVersion getExtOrDefault('buildToolsVersion')

**Comment by karuzo17** [comment]
I am having the same issue. Any recommendations?

**Comment by zihejia** [comment]
Thanks @Qdafengzi and @karuzo17 for bring it up. It's fixed in 3.0.6.
